inlong-manager/manager-dao/src/main/resources/mappers/SortClusterMapper.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
+<mapper namespace="org.apache.inlong.manager.dao.mapper.SortClusterMapper">
+    <resultMap id="SortTaskDTO" type="org.apache.inlong.manager.dao.entity.SortTaskDTO">
+        <result column="inlong_cluster_name" jdbcType="VARCHAR" property="sortClusterName"/>
+        <result column="sort_task_name" jdbcType="VARCHAR" property="sortTaskName"/>
+        <result column="sort_consumer_group" jdbcType="VARCHAR" property="sortConsumerGroup"/>
+        <result column="sink_type" jdbcType="VARCHAR" property="sinkType"/>
+        <result column="data_node_name" jdbcType="VARCHAR" property="dataNodeName"/>
+    </resultMap>
+    <resultMap id="SortSinkParamsDTO" type="org.apache.inlong.manager.dao.entity.SortSinkParamsDTO">
+        <result column="sort_task_name" jdbcType="VARCHAR" property="sortTaskName"/>
+        <result column="name" jdbcType="VARCHAR" property="name"/>
+        <result column="type" jdbcType="VARCHAR" property="type"/>
+        <result column="ext_params" jdbcType="VARCHAR" property="extParams"/>
+    </resultMap>
+    <resultMap id="SortIdParamsDTO" type="org.apache.inlong.manager.dao.entity.SortIdParamsDTO">
+        <result column="sort_task_name" jdbcType="VARCHAR" property="sortTaskName"/>
+        <result column="inlong_group_id" jdbcType="VARCHAR" property="inlongGroupId"/>
+        <result column="inlong_stream_id" jdbcType="VARCHAR" property="inlongStreamId"/>
+        <result column="ext_params" jdbcType="VARCHAR" property="extParams"/>
+    </resultMap>
+
+    <select id="selectAllTasks" resultMap="SortTaskDTO">
+        select
+            inlong_cluster_name, sort_task_name, sort_consumer_group, sink_type, data_node_name
+        from stream_sink
+        <where>
+            is_deleted = 0
+        </where>
+        group by
+            inlong_cluster_name, sort_task_name, sort_consumer_group, sink_type, data_node_name
+    </select>
+
+    <select id="selectAllSinkParams" resultMap="SortSinkParamsDTO">
+        select
```

Comment:
Why not combine the `selectAllSinkParams` and `selectAllIdParams` into one query through the associated query?